api/client/annotations/export_test.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2022 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package annotations
+
+import "github.com/juju/juju/api/base"
+
+func NewClientFromCaller(caller base.FacadeCaller) *Client {
```

Comment:
Prefer to put _package_test.go_ into the annotations package and add this method there. We'd like to strangle out all the export_test files.